docs/api/dt/count.rst

```diff
@@ -1,16 +1,17 @@
 
 .. xfunction:: datatable.count
-    :src: src/core/expr/head_reduce_unary.cc count_reducer
+    :src: src/core/expr/fexpr_count_countna.cc pyfn_count
     :cvar: doc_dt_count
     :tests: tests/test-reduce.py
-    :signature: count(cols)
+    :signature: count(cols=None)
 
-    Calculate the number of non-missing values for each column from `cols`.
+    Calculate the number of non-missing values for each column from `cols`, if `cols` is provided, 
+    or the total number of rows if `cols` is not provided.
 
     Parameters
     ----------
     cols: FExpr
-        Input columns.
+        Input columns. If no `cols` is passed, then the count of all rows is returned.
```

Comment:
```suggestion
        Input columns if any.
```

Reply:
We don't need to mention here what is returned, as this is not a `return:` section.